.github/workflows/ci.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2019 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+
+  build-chart:
+    runs-on: ubuntu-latest
+    needs: check-for-chart-changes
```

Comment:
does this mean that a *check-for-chart-changes* action is required to run first? If so, where is it defined?